packages/myst-directives/src/blockquote.ts

```diff
@@ -0,0 +1,50 @@
+import type { DirectiveSpec, DirectiveData, GenericNode } from 'myst-common';
+import { normalizeLabel } from 'myst-common';
+import type { Container } from 'myst-spec-ext';
+import classNames from 'classnames';
+
+export const blockQuoteDirective: DirectiveSpec = {
+  name: 'block-quote',
+  alias: ['epigraph', 'pull-quote'],
+  doc: 'Block quotes are used to indicate that the enclosed content forms an extended quotation. They may be followed by an inscription or attribution formed of a paragraph beginning with `--`, `---`, or an em-dash.',
+  options: {
+    label: {
+      type: String,
+      alias: ['name'],
+    },
+    class: {
+      type: String,
+      doc: `CSS classes to add to your block-quote. Special classes include:
+
+- \`pull-quote\`: used for block-quotes which should attract attention
+- \`epigraph\`: used for block-quotes that are usually found at the beginning of a document`,
+    },
+  },
+  body: {
+    type: 'myst',
+    doc: 'The body of the block-quote.',
```

Comment:
```suggestion
    doc: 'The body of the quote.',
```

as it might be a block/pull/epigraph